lib/Parser.php

```diff
@@ -236,16 +236,12 @@ public function getDocument() : Document
         return $this->document;
     }
 
-    public function includeFileAllowed(string $path) : bool
+    private function includeFileAllowed(string $path) : bool
```

Comment:
I'm under the impression that we're still fixing/improving this lib, so we aren't protecting BC. This method isn't used anywhere else in this project. I can also move this down to the bottom (below the public/protected functions) - I didn't want to make the diff unnecessarily big.